### PR TITLE
[ios] Replace matrix icon

### PR DESCRIPTION
**Pull request by itfarrier**
Replace matrix icon based on https://github.com/organicmaps/organicmaps/issues/8069.

Screenshots with old and new icons:
old
new

Fix: #8069

### CODE REVIEW

**Comment by RedAuburn** [comment]
> I don't see any difference...

the 'M' in the old icon is slightly off-centre (to the right)

File: iphone/Maps/Images.xcassets/About/ic_social_media_matrix.imageset/Contents.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "ic_social_media_matrix.pdf",
```
Comment (kirylkaveryn) [inline iphone/Maps/Images.xcassets/About/ic_social_media_matrix.imageset/Contents.json:4]: In some cases it is safer to use pdfs for the image assets in iOS. Let's use the same format as for the others social media icons.

Comment (itfarrier) [reply iphone/Maps/Images.xcassets/About/ic_social_media_matrix.imageset/Contents.json]: @kirylkaveryn, hello! I've replaced the file using PDF. Could you please explain in which cases we use PDFs? I see from [Apple's Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/icons) that we can use both PDF and SVG, but there is no information about compatibility or other details.

Comment (kirylkaveryn) [reply iphone/Maps/Images.xcassets/About/ic_social_media_matrix.imageset/Contents.json]: Usually, ios draws svgs and pdfs well, but sometimes (not usually) it fails to draw the complex svg gradients with paths.
In this particular case, it is just to use a consistent format for the similar.

**Review by biodranik — APPROVED** [review]
Thanks! There is also fosstodon icon in svg. 

How do you create pdf from svg? It would be easier to have everything in SVG to be easily reused on other platforms... @kirylkaveryn

**Comment by itfarrier** [comment]
> Thanks! There is also fosstodon icon in svg.
> 
> How do you create pdf from svg? It would be easier to have everything in SVG to be easily reused on other platforms... @kirylkaveryn

@biodranik, hello, I've used Inkscape for translate SVG to PDF.

**Pull request merged**